mkosi

```diff
@@ -2887,7 +2954,13 @@ def make_tar(args: CommandLineArguments,
 
     with complete_step('Creating archive'):
         f: BinaryIO = cast(BinaryIO, tempfile.NamedTemporaryFile(dir=os.path.dirname(args.output), prefix=".mkosi-"))
-        run(["tar", "-C", root, "-c", "-J", "--xattrs", "--xattrs-include=*", "."],
+# OpenMandriva defaults to bsdtar(libarchive) which uses POSIX argument list so let's keep a separate list
+        if (shutil.which('bsdtar') and Distribution.openmandriva):
```

Comment:
```suggestion
        if shutil.which('bsdtar') and args.distribution == Distribution.openmandriva:
```
Otherwise the second half of the conditional is always truthy because the value of Distribution.openmandriva is 10. Also, the parentheses are uncommon in Python for single-line statements.

Reply:
Thanks for catching this.